bundles/core/org.eclipse.smarthome.core.thing/ESH-INF/config/config.xml

```diff
@@ -16,5 +16,35 @@
             <default>true</default>
         </parameter>
     </config-description>
+    
+    <config-description uri="system:firmware-status-info-job">
+        <parameter name="period" type="integer" min="1" required="true">
+            <label>Period</label>
+            <description><![CDATA[
```

Comment:
You do not need a CDATA section and the `<p>` elements here, since you only have a simply sentence as a description. Same for the parameters below.